### PR TITLE
New kit updates

### PR DESCRIPTION
**Pull request by priyaprafful**
Updates
1. Updated functions and templating based on new client and helpers kit.
2. Created SliceZone.
3. Changed templated from pug to ejs

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,106 +1,77 @@
...
-    }
+// Route for page
+route.get(
+  '/:uid',
+  asyncHandler(async (req, res, next) => {
+    const uid = req.params.uid;
+    const pageContent = await client.getByUID('page', uid);
+    res.render('page', { pageContent });
   })
-  .catch((error) => {
```
Comment (faresd) [inline app.js:103]: @priyaprafful Why you don't want to log the error anymore to next.js console?

File: config.js [hunk 2]
```diff
@@ -1,30 +1,32 @@
...
  */
-var express = require('express'),
-    favicon = require('serve-favicon'),
-    logger = require('morgan'),
-    bodyParser = require('body-parser'),
-    methodOverride = require('method-override'),
-    errorHandler = require('errorhandler'),
-    http = require('http'),
-    path = require('path');
+import express from 'express';
```
Comment (faresd) [inline config.js:4]: Probably you need to add a js linter to the project as eslint 
https://nextjs.org/docs/basic-features/eslint

Comment (faresd) [reply config.js:4]: Normally semicolons are not recommended in imports.

File: views/SliceZone.ejs [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    else if (slice.slice_type === 'image_highlight') {
+      return prevSlices + include('./slices/ImageHighlight.ejs', { slice: slice })
+    }
+    else {
+      return '<h1>No Slices found</h1>';
+    } 
+  }, '')
+}) %>
+
+<%- getSlices(SliceZone) %>
```
Comment (faresd) [inline views/SliceZone.ejs:24]: I don't understand this

File: views/page.ejs [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<html>
+  <head>
+    <%- include('./partials/head'); %>
+  </head>
+  <body>
+    <%- include('./partials/navigation'); %>
+    <div class="container">
+      <%- include("SliceZone", {SliceZone: pageContent.data.page_content}); %>
```
Comment (faresd) [inline views/page.ejs:8]: why don't you pass SliceZone directly?

File: config.js (hunk 2 above)
Comment (priyaprafful) [reply config.js:4]: Added eslint

File: views/SliceZone.ejs (hunk 3 above)
Comment (priyaprafful) [reply views/SliceZone.ejs:24]: Removed function

**Pull request merged**